content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,130 +2,181 @@
 their user account settings.
 
-.. note::
-   If an employee has time off scheduled in the :guilabel:`Time Off` application, they will not be
-   assigned tickets during that time. If no employees are available, the system will look ahead
-   until there is a match.
+.. important::
+   If an employee has time off scheduled in the *Time Off* application, they are **not** assigned
+   tickets during that time. If no employees are available, the system looks ahead until there is a
+   match.
 
 .. seealso::
    - :ref:`Manage users <users/add-individual>`
    - :doc:`Access rights </applications/general/users/access_rights>`
 
-Create or modify kanban stages
-==============================
+Create or modify stages
+=======================
 
-:guilabel:`Stages` are used to organize the *Helpdesk* pipeline and track the progress of tickets.
-Stages are customizable, and can be renamed to fit the needs of each team.
-
-To view or modify *Helpdesk* stages, go to :menuselection:`Helpdesk --> Configuration --> Stages`.
+*Stages* are used to organize the *Helpdesk* pipeline and track the progress of tickets. Stages are
+customizable, and can be renamed to fit the needs of each team.
 
 .. important::
-   :ref:`Developer mode <developer-mode>` must be activated in order to access the stages menu. To
-   activate developer mode go to :menuselection:`Settings --> General Settings --> Developer Tools`
-   and click on :guilabel:`Activate the developer mode`.
+   :ref:`Developer mode <developer-mode>` **must** be activated to access the stages menu. To
+   activate developer mode, go to :menuselection:`Settings app --> General Settings --> Developer
+   Tools`, and click :guilabel:`Activate the developer mode`.
 
-The list view shows an overview of all the stages currently available in Helpdesk. They are listed
-in the order they appear in the pipeline. To change the order of the stages, use the arrow buttons
-on the left side of the list.
+To view or modify *Helpdesk* stages, go to :menuselection:`Helpdesk app --> Configuration -->
+Stages`.
 
-.. tip::
-   Change the stage order on the kanban view by dragging and dropping individual columns.
+The default list view on the :guilabel:`Stages` page displays the stages currently available in
+*Helpdesk*. They are listed in the order they appear in the pipeline.
 
-.. image:: getting_started/stages-create-new.png
+To change the order of the stages, click the :guilabel:`six square` icon, to the left of the stage
+name, and drag it to the desired place on the list.
+
+.. image:: getting_started/stages-list-buttons.png
    :align: center
-   :alt: View of the stage list page emphasizing the option to create a new stage
+   :alt: View of the stage list page emphasizing the buttons used to change the order the stages
+         appear in the list.
+
+.. tip::
+   Change the stage order on the Kanban view by dragging and dropping individual columns.
```

Comment:
I'm not entirely sure that the Kanban view of the Stages page in Helpdesk is in columns. At the moment, I'm only seeing Kanban cards of the stages -- no columns. This is worth a double-check. And, if you're encountering the same thing as I am, I would suggest removing this tip admonition.

Reply:
UPDATE: Consider specifying that you are referencing the Tickets Kanban columns here. Could be helpful!